src/compiler/lsp/completion.cc

```diff
@@ -369,14 +369,14 @@ void CompletionHandler::import_path(const char* path,
                                     Filesystem* fs) {
   if (is_first_segment) {
     current_package.list_prefixes([&](const std::string& candidate) {
-      complete(candidate.c_str(), CompletionKind::MODULE);
+      complete(candidate, CompletionKind::MODULE);
```

Comment:
Changes in this file don't change memory behavior. They are just cleaner this way.